mainmatter/introduction/index.tex

```diff
@@ -26,7 +30,7 @@ \chapter{Introduction}\label{ch:introduction}
 The aim is to generate insight into the graphics \glspl{API}; Direct3D 12 and Vulkan, and to generate tools for future developers that can be used to evaluate and improve their \glspl{API}.
 
 Direct3D 12 and Vulkan are chosen, because they are released close to eachother (2015 and 2016 respecively), and have some overlapping ideas on how to improve graphics \glspl{API}, such as better multithread support on the CPU.
-The predesessors to these \glspl{API} (Direct3D 11 and OpenGL 4.5) are some of the most broadly used graphics \glspl{API} for realtime rendering, and we suspect that Direct3D 12 and Vulkan will have similar success.
+The predesessors to these \glspl{API} (Direct3D 11 and OpenGL 4.5) are some of the most broadly used graphics \glspl{API} for realtime rendering, and we suspect that Direct3D 12 and Vulkan will have similar success in the comming years.
```

Comment:
Er ikke helt sikker på at jeg er enig med denne ændring, men det er bare min mening

Reply:
The change stays